### PR TITLE
Issue #1255 | Expanding of the benefits of the Company Accounts

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) :
- Fixes the https://github.com/magento/merchdocs/issues/1255
- Describes what is Company Accounts
- Describes the benefits of the Company Accounts
- Adds useful links to other topics 

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/customers/account-companies.html

whatsnew
Updated [Company Accounts](https://docs.magento.com/user-guide/customers/account-companies.html) to describe the benefits and feature capabilities for using B2B company accounts.

### CODE REVIEW

File: src/customers/account-companies.md [hunk 1]
```diff
@@ -3,12 +3,26 @@ b2b_only: true
...
+Magento 2 Company Account transforms a default [individual] ({% link customers/account-create.md %}) customer account to the company accounts and supports customers to create and manage company accounts.
 
-Company accounts can be set up from the storefront or from the Admin. By default, the ability to create company accounts from the storefront is enabled. If allowed in the configuration, a visitor to the store can request to open a company account. After the company account is approved, the company administrator can set up the company structure and users with various levels of permission.
+By introducing B2B company accounts to your Magento 2 store, you can simplify the corporate shopping, let companies create multiple sub-accounts with flexible permissions based on user roles in the organization. Depending on the customer groups of the company, the store owner can adjust promotions and prices to suit their needs. It allows admins to create highly customized offers to cater to the shoppers’ demands and get more orders.
```
Comment (leslietilling) [inline src/customers/account-companies.md:8]:
```suggestion
When you incorporate B2B company accounts in your Magento store, you can simplify the corporate shopping by enabling companies to create multiple sub-accounts with flexible permissions based on user roles in their organization. Depending on the customer groups of the company, a store administrator can adjust promotions and prices to suit their needs, and create highly customized offers that cater to the shoppers’ demands and increase orders.
```

File: src/customers/account-companies.md [hunk 2]
```diff
@@ -3,12 +3,26 @@ b2b_only: true
...
+
+Advantage of Magento 2 Company Account:
```
Comment (leslietilling) [inline src/customers/account-companies.md:10]:
```suggestion
Advantages of  a company account:
```

File: src/customers/account-companies.md [hunk 3]
```diff
@@ -3,12 +3,26 @@ b2b_only: true
...
+
+1. [B2B]({% link configuration/general/b2b-features.md  %}) Company Account offers unlimited users and the creation of additional accounts, thereby simplifying corporate purchases.
```
Comment (leslietilling) [inline src/customers/account-companies.md:12]: This should really be an unordered list
```suggestion
1. - A [B2B]({% link configuration/general/b2b-features.md  %}) company account offers unlimited users and the creation of additional accounts, thereby simplifying corporate purchases.
```

File: src/customers/account-companies.md [hunk 4]
```diff
@@ -3,12 +3,26 @@ b2b_only: true
...
+
+1. There is an ability to build a smart B2B company account hierarchy with different [roles and permissions] ({% link customers/account-company-roles-permissions.md %}) as placing orders.
```
Comment (leslietilling) [inline src/customers/account-companies.md:14]:
```suggestion
- Includes support for a "smart" company account hierarchy with different [roles and permissions] ({% link customers/account-company-roles-permissions.md %}) for placing orders.
```

File: src/customers/account-companies.md [hunk 5]
```diff
@@ -3,12 +3,26 @@ b2b_only: true
...
+
+1. Allows to increase income by offering a [Company Store Credits] ({% link customers/account-dashboard-company-credit.md %}) payment method.
```
Comment (leslietilling) [inline src/customers/account-companies.md:16]:
```suggestion
- Provides a mechanism for merchants to increase income by offering [company store credit] ({% link customers/account-dashboard-company-credit.md %}) as a payment method.
```

File: src/customers/account-companies.md [hunk 6]
```diff
@@ -3,12 +3,26 @@ b2b_only: true
...
+
+1. Permit [managing] ({% link customers/account-company-manage.md %}) all Company Accounts in Magento Admin.
```
Comment (leslietilling) [inline src/customers/account-companies.md:18]:
```suggestion
1. Supports the [management] ({% link customers/account-company-manage.md %}) of all company accounts in the Admin.
```

File: src/customers/account-companies.md [hunk 7]
```diff
@@ -3,12 +3,26 @@ b2b_only: true
...
 
 ## View company accounts
 
+The Companies grid lists all active company accounts and pending requests, regardless of status setting. It also provides the tools needed to create and [manage]({% link customers/account-company-manage.md %}) company accounts. Use the standard [grid controls]({% link stores/admin-grid-controls.md %}) to filter the list, and adjust the [column layout]({% link stores/admin-grid-layout.md %}). For a list of column descriptions, see the Column Descriptions in [Managing Company Accounts]({% link customers/account-company-manage.md %}).
```
Comment (leslietilling) [inline src/customers/account-companies.md:22]:
```suggestion
The _Companies_ grid lists all active company accounts and pending requests, regardless of status setting. It also provides the tools needed to [create]({% link customers/account-company-create.md %}) and [manage]({% link customers/account-company-manage.md %}) company accounts. Use the standard [grid controls]({% link stores/admin-grid-controls.md %}) to filter the list, and adjust the [column layout]({% link stores/admin-grid-layout.md %}). For a list of column descriptions, see the _Column Descriptions_ section in [Managing Company Accounts]({% link customers/account-company-manage.md %}).
```

File: src/customers/account-companies.md [hunk 8]
```diff
@@ -3,12 +3,26 @@ b2b_only: true
...
+
+Company accounts can be set up from the storefront or from the Admin. By default, the ability to create company accounts from the storefront is enabled. If allowed in the configuration, a visitor to the store can request to open a company account. After the company account is approved, the company administrator can set up the company structure and users with various levels of permission.
```
Comment (leslietilling) [inline src/customers/account-companies.md:24]:
```suggestion
Customers can create a new company accounts from the storefront, or a merchant can create one from the Admin. By default, the ability to create company accounts from the storefront is enabled. If allowed by the configuration, a visitor to the store can request to open a company account. After the company account is approved, the company administrator can set up the company structure and users with various levels of permission.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just some edits for style and clarity

**Comment by EvheniiaKolkhiieva** [comment]
Hello @leslietilling , thank you a lot for reviewing.

I have implemented your suggestions, could you please approve this PR? 
Thank you 😊

File: src/customers/account-companies.md (hunk 8 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-companies.md]: Done ✅

File: src/customers/account-companies.md (hunk 7 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-companies.md]: Done ✅

File: src/customers/account-companies.md (hunk 6 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-companies.md]: Done ✅

File: src/customers/account-companies.md (hunk 5 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-companies.md]: Done ✅

File: src/customers/account-companies.md (hunk 4 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-companies.md]: Done ✅

File: src/customers/account-companies.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-companies.md]: Done ✅

File: src/customers/account-companies.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-companies.md]: Done ✅

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**